git_machete/options.py

```diff
@@ -44,14 +44,14 @@ def validate(self) -> None:
         if self.opt_as_root and self.opt_onto:
             raise MacheteException(
                 "Option `-R/--as-root` cannot be specified together with `-o/--onto`.")
-        if self.opt_no_edit_merge and not self.opt_merge:
-            raise MacheteException(
-                "Option `--no-edit-merge` only makes sense when using merge and "
-                "must be specified together with `-M/--merge`.")
-        if self.opt_no_interactive_rebase and self.opt_merge:
-            raise MacheteException(
-                "Option `--no-interactive-rebase` only makes sense when using "
-                "rebase and cannot be specified together with `-M/--merge`.")
+        # if self.opt_no_edit_merge and not self.opt_merge:
```

Comment:
To be reverted? 🤔

Reply:
@PawelLipski well, it would depend on whether we allow `merge=no`` or  ``merge=yes``

Reply:
Both `merge=...` to be dropped